tests/commands/test_dump_data_to_clickhouse.py

```diff
@@ -76,7 +80,16 @@ class DummySink(ModelBaseSink):
     factory = dummy_model_factory()
 
     def get_queryset(self, start_pk=None):
-        return [self.factory(id) for id in range(1, 5)]
+        qs = MockSet(
+            MockModel(mock_name='john', email='john@edx.com', pk=1),
```

Comment:
Last thing, I think - can we not use edx.com email addresses here? I usually use something like `john@test.invalid`.